packages/manager/modules/sms/src/sms/batches/telecom-sms-batches.controller.js

```diff
@@ -0,0 +1,53 @@
+export default class {
+  /* @ngInject */
+  constructor($translate) {
+    this.$translate = $translate;
+  }
+
+  $onInit() {
+    this.actions = [
```

Comment:
Copy paste of https://github.com/ovh/manager/blob/master/packages/manager/modules/sms/src/sms/sms/telecom-sms-sms.controller.js#L9 (with links modifications)